src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -34,7 +37,6 @@ properties:
   `6831`).
 * `MaxPayloadSizeInBytes`: The maximum size of each UDP packet that gets
   sent to the agent. (default `4096`).
-* `ProcessTags`: Which tags should be sent with telemetry.
```

Comment:
this is the most important line of the PR I think, since this parameter is no longer used in options.